students/sangwoong/users/views.py

```diff
@@ -56,14 +58,20 @@ def post(self, request):
             data          = json.loads(request.body)
             user_account  = data["account"]
             user_password = data["password"]
+            user_id       = User.objects.get(account = user_account).id
 
             if not User.objects.filter(account = user_account).exists() :
                 return JsonResponse({"Message": "INVALIDE_USER"}, status=401)
 
-            if User.objects.get(account = user_account).password != user_password :
-                return JsonResponse({"Message": "INVALID_USER"}, status=401)
+            hashed_password = user_password.encode("utf-8")
+            saved_password  = User.objects.get(account = user_account).password
```

Comment:
user의 존재 여부를 위의 if문에서 확인 했기 때문에 이제는 ORM을 통해서 user 정보를 가져 오면 됩니다. 
정보를 가져올 때도 필드 값 하나 하나를 가져오는 것이 아니고, user 라는 변수에 User data를 담아두고 사용하면 매번 ORM으로 user 정보를 요청하지 않아도 됩니다.

```suggestion
            user = User.objects.get(account=user_account)
            
            hashed_password = user_password.encode("utf-8")
            saved_password  = user.password
```